daffodil-runtime1/src/main/scala/org/apache/daffodil/runtime1/processors/EvEscapeSchemes.scala

```diff
@@ -114,7 +123,12 @@ class EscapeSchemeCharUnparseEv(
   def compute(state: ParseOrUnparseState) = {
     val escChar = escapeChar.evaluate(state).charAt(0)
     val optEscEscChar = evalAndConvertEEC(state)
-    new EscapeSchemeCharUnparserHelper(escChar, optEscEscChar, extraEscapedCharsCooked, ci)
+    new EscapeSchemeCharUnparserHelper(
+      escChar,
+      optEscEscChar,
+      extraEscapedCharsCooked(state),
```

Comment:
Just to make our code more consistent with escEscChar, can you move this up and change the name, something like:

```scala
val extraEscChars = evalAndConvertExtraEscapedChars(state)
new EscpeSchemeCharUnparserHalper(..., extraEscChars, ...)
```

Same with below use.